.github/workflows/ci-windows.yml

```diff
@@ -68,3 +68,48 @@ jobs:
           chdman.exe
           unidasm.exe
         if-no-files-found: error
+
+  build-windows-arm64:
+    runs-on: windows-11-arm
+    defaults:
+      run:
+        shell: msys2 {0}
+    strategy:
+      matrix:
+        subtarget: [mame, tiny]
+        include:
+          - subtarget: mame
+            executable: mame
+          - subtarget: tiny
+            executable: mametiny
```

Comment:
This should be integrated into the matrix with the other builds.

There’s also no point building the tiny subtarget with the same configuration as the full build.  We build the tiny subtarget with clang for x86-64 to test that the Windows-specific parts of MAME can be built with clang without generating too much load on the CI servers.

Reply:
Ahhhhhhhh ok . I had no idea.